src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -583,8 +668,11 @@ public Response corsPreflightMetricDelete(@HeaderParam("Access-Control-Request-H
 	@DELETE
 	@Produces(MediaType.APPLICATION_JSON + "; charset=UTF-8")
 	@Path("/metric/{metricName}")
-	public Response metricDelete(@PathParam("metricName") String metricName) throws Exception
+	public Response metricDelete(@Context HttpHeaders httpHeaders, @PathParam("metricName") String metricName) throws Exception
 	{
+		Scope scope = activateScope("datapoints_" + metricName, httpHeaders);
```

Comment:
I think this naming is [not recommended (too specific case)](https://github.com/opentracing/specification/blob/master/specification.md#start-a-new-span)

Adding the ``metricName`` as a tag makes more sense here.